lightstepoc/options.go

```diff
@@ -60,6 +60,15 @@ func WithComponentName(componentName string) Option {
 	}
 }
 
+// WithDefaultTags sets tags that will be appended to every span that is exported to the trace.
```

Comment:
I think we should rename this to `WithDefaultAttributes` to be consistent with OpenCensus's naming. ("Tag" means something else to them; "attribute" is more akin to OpenTracing's tags.)